### DISCUSSION TITLE
Please A tutorial or just help for downloading crunchyroll

### DISCUSSION THREAD

**Discussion by KuroProjects** [anidl/multi-downloader-nx#319] [Q&A]
i want to download episodes from crunchyroll and i followed all the steps but i am still getting errors 
please someone help

**Comment by izu-co** [comment]
Hey there. I know it can be a bit overwhelming, but maybe we can start by you stating what you accomplished and where you are currently stuck. Please, provide screenshots and tell me what you are trying to do.

**Comment by KuroProjects** [comment]
hi
so basically i am trying this tool in shell
but it is giving this problem everytime i try it

![image](https://user-images.githubusercontent.com/92580272/174476167-30d0dee4-97e1-4212-92ab-3244cc56e7b5.png)

**Reply by izu-co** [reply]
You need to update your nodejs version

**Reply by yagmur888** [reply]
Please tell me how did you even get to this point. PLEASE!
